app/models/enterprise_fee.rb

```diff
@@ -68,6 +69,16 @@ def ensure_valid_tax_category_settings
     return true
   end
 
+  def ensure_no_product_distributions
+    dependent_distribution = ProductDistribution.where(enterprise_fee_id: self).first
+    if dependent_distribution
```

Comment:
CodeClimate is complaining about this line. It's as Luis pointed out the other day, we can use `return true unless dependent_distribution` here, remove the `end` below, and remove the indentation on the lines in between.

Reply:
yep, although if you do it here you would be moving the error case to the base indentation which is not so obviously good, you want the base indentation to be the happy path of the method (anyway, it's a small detail, I am just commenting because I think it's interesting)